integration-test/spark3/src/test/java/org/apache/uniffle/test/GetShuffleReportForMultiPartTest.java

```diff
@@ -145,12 +145,12 @@ public void resultCompareTest() throws Exception {
   Map runTest(SparkSession spark, String fileName) throws Exception {
     Thread.sleep(4000);
     Map<Integer, String> map = Maps.newHashMap();
-    Dataset<Row> df2 = spark.range(0, 1000, 1, 10)
+    Dataset<Row> df2 = spark.range(0, 10000, 1, 10)
         .select(functions.when(functions.col("id").$less(250), 249)
             .otherwise(functions.col("id")).as("key2"), functions.col("id").as("value2"));
-    Dataset<Row> df1 = spark.range(0, 1000, 1, 10)
+    Dataset<Row> df1 = spark.range(0, 10000, 1, 10)
         .select(functions.when(functions.col("id").$less(250), 249)
-            .when(functions.col("id").$greater(750), 1000)
+            .when(functions.col("id").$greater(750), 10000)
```

Comment:
Why do we change this?

Reply:
The data size of zstd compressed is slow than the skew join threshold. So increase this record number

Reply:
If we use lz4 as default compress codec, it seems that we don't need modify this.

Reply:
Yes.